### PR TITLE
allow multiple targets in ipmi stanza, change ipmi poll rate

### LINKED ISSUE #175: IPMI ingester config should support multiple targets per block
most deployments will have a shared credential across many machines, blocks should support multiple targets.

Turn this: https://github.com/gravwell/gravwell/blob/dev/ingesters/IPMIIngester/config.go#L25 into a `[]string` so that i can have a list of targets.  Forcing a user to specify a whole new block with creds for every machine is tedious.

This way if you have 3 "enclaves" you can specify 3 blocks with a set of target IPs in each rather than a whole new block for every single machine.

### LINKED ISSUE #182: make ipmi poll rate 1 minute

### PR DESCRIPTION
**Pull request by djfritz**
Fixes #175
Fixes #182

### CODE REVIEW

File: ingesters/IPMIIngester/main.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ var (
 	ipmiConns map[string]*handlerConfig
 )
 
-const PERIOD = 10 * time.Second
+const PERIOD = time.Minute
```
Comment (traetox) [inline ingesters/IPMIIngester/main.go:52]: any reason to make this configurable ?

**Review by traetox — COMMENTED** [review]
i think having a solid default of 10 minutes is good, but it should be configurable.

File: ingesters/IPMIIngester/main.go (hunk 1 above)
Comment (djfritz) [reply ingesters/IPMIIngester/main.go:52]: agreed - update incoming

**Comment by djfritz** [comment]
@traetox PTAL - also added description field on certain messages

**Review by traetox — APPROVED** [review]
LGTM

**Pull request merged**